app/src/main/java/com/endava/myendava/fragments/FaqFragment.java

```diff
@@ -62,7 +77,11 @@ public View provideFragmentView(LayoutInflater inflater, ViewGroup parent, Bundl
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
```

Comment:
mb would be nice to break these into smaller pieces so onViewCreated won't be that large (just a thought :) )